packages/core/src/render3/instructions.ts

```diff
@@ -134,13 +134,15 @@ export function setHostBindings(tView: TView, viewData: LView): void {
 }
 
 /** Refreshes content queries for all directives in the given view. */
-function refreshContentQueries(tView: TView): void {
+function refreshContentQueries(tView: TView, lView: LView): void {
   if (tView.contentQueries != null) {
     setCurrentQueryIndex(0);
     for (let i = 0; i < tView.contentQueries.length; i++) {
       const directiveDefIdx = tView.contentQueries[i];
       const directiveDef = tView.data[directiveDefIdx] as DirectiveDef<any>;
-      directiveDef.contentQueriesRefresh !(directiveDefIdx - HEADER_OFFSET);
+      const directiveIndex = directiveDefIdx - HEADER_OFFSET;
+      directiveDef.contentQueries !(
+          RenderFlags.Update, loadInternal(lView, directiveIndex), directiveDefIdx);
```

Comment:
`loadInternal` just adds the `HEADER_OFFSET` back. Since we already have a raw index, seems like we should use it directly (and remove extra var).

```suggestion
          RenderFlags.Update, lView[directiveDefIndex], directiveDefIdx);
```